config/constants.yml

```diff
@@ -25,9 +25,8 @@ TERMINUS_TIME_ZONE:     'UTC'
 
 # File Storage
 TERMINUS_CACHE_DIR:     '~/.terminus/cache'
-TERMINUS_CONFIG_DIR:    '~/terminus'
 TERMINUS_LOG_DIR:       '/tmp'
-TERMINUS_PLUGINS_DIR:   '[[ TERMINUS_CONFIG_DIR ]]/plugins'
+TERMINUS_PLUGINS_DIR:   '~/.terminus/plugins'
```

Comment:
Yeah. I'm of a similar mindset, but we decided that since the ultimate goal is to have the plugins all managed by terminus then it was probably ok to consider the files themselves non-user facing (even if they are for now). This way we're not creating multiple directories in the user's home dir.